indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -57,6 +61,88 @@ def _process(self):
+            if column_name in available_columns:
+                enabled_columns.append({'id': column_name, 'title': available_columns[column_name]})
+                del available_columns[column_name]
+        for column_name, column_title in available_columns.iteritems():
+            disabled_columns.append({'id': column_name, 'title': column_title})
+        disabled_columns.sort(key=itemgetter('title'))
+
+        available_forms = {regform.id: regform for regform in regforms}
+        enabled_forms = []
+        disabled_forms = []
+        # Handle forms that have already been sorted by the user.
+        for form_id in registration_settings.get_participant_list_form_ids(self.event):
+            try:
+                regform = available_forms[form_id]
+            except KeyError:
+                continue
+            # Make sure publication was not disabled since the display settings were modified.
+            if regform.publish_registrations_enabled:
+                enabled_forms.append(regform)
+                del available_forms[form_id]
+        for form_id, regform in available_forms.iteritems():
+            # There might be forms with publication enabled that haven't been sorted by the user yet.
+            if regform.publish_registrations_enabled:
+                enabled_forms.append(regform)
+            else:
+                disabled_forms.append(regform)
+        disabled_forms.sort(key=attrgetter('title'))
+
+        merge_forms = registration_settings.get(self.event, 'merge_registration_forms')
+
+        return WPManageRegistration.render_template('management/regform_display.html', self.event, event=self.event,
+                                                    regforms=regforms, enabled_columns=enabled_columns,
+                                                    disabled_columns=disabled_columns, enabled_forms=enabled_forms,
+                                                    disabled_forms=disabled_forms, merge_forms=merge_forms, form=form)
+
+
+class RHManageRegistrationFormDisplay(RHManageRegFormBase):
+    """Choose the columns to be shown on the participant list for a particular form"""
+
+    def _process(self):
+        form = ParticipantsDisplayFormColumnsForm()
+        if form.validate_on_submit():
+            registration_settings.set_participant_list_columns(self.event, form.json.data['columns'], self.regform)
+            return jsonify_data(flash=False)
+
+        available_fields = {field.id: field for field in self.regform.active_fields}
+        enabled_fields = []
+        for field_id in registration_settings.get_participant_list_columns(self.event, self.regform):
+            try:
+                field = available_fields[field_id]
+            except KeyError:
+                continue
+            enabled_fields.append(field)
+            del available_fields[field_id]
+        disabled_fields = available_fields.values()
```

Comment:
i'd move this down one line (i.e. have the empty line before this one instead of after)